manim/mobject/svg/text_mobject.py

```diff
@@ -598,7 +598,28 @@ class PangoText(SVGMobject):
         The mobject like :class:`.VGroup`.
 
     Examples
-    --------
+    ---------
+    .. manim:: MultipleFonts
+        :save_last_frame:
+
+        class MultipleFonts(Scene):
+            def construct(self):
+                morning = PangoText("வணக்கம்", font="sans-serif")
+                chin = PangoText(
+                    "見 角 言 谷  辛 辰 辵 邑 酉 釆 里!", t2c={"見 角 言": BLUE}
+                )  # works same as ``Text``.
+                mess = PangoText("Multi-Language", style=BOLD)
+                russ = PangoText("Здравствуйте मस नम म ", font="sans-serif")
+                hin = PangoText("नमस्ते", font="sans-serif")
+                arb = PangoText(
+                    "صباح الخير \n تشرفت بمقابلتك", font="sans-serif"
```

Comment:
This line \n  makes problems with building the docs.
Anyone an idea to solve this?
Maybe @behackl ?

Reply:
Ahh, I see: the docstring should be marked as a raw string (add `r` before `"""` in Line585, https://github.com/ManimCommunity/manim/pull/533/files#diff-cf9247b89e1b4debba20823314a969f8R585). That should fix it. :-)

Reply:
perfect!